parm/config/config.resources

```diff
@@ -129,6 +129,7 @@ elif [ $step = "anal" ]; then
     export npe_anal=1000
     export nth_anal=5
     export npe_anal_gfs=1000
+    export nth_anal_gfs=$nth_anal
```

Comment:
I don't think you need this line, since Line 147 will take care of that.

Reply:
removed